lib/Cro/WebSocket/Client.pm6

```diff
@@ -45,7 +45,7 @@ class Cro::WebSocket::Client {
         @!headers = Cro::HTTP::Header.new(name => 'Upgrade', value => 'websocket'),
                     Cro::HTTP::Header.new(name => 'Connection', value => 'Upgrade'),
                     Cro::HTTP::Header.new(name => 'Sec-WebSocket-Version', value => '13');
-        without @headers.first(*.name eq 'Sec-WebSocket-Protocol') {
+        without @headers.first(*.name.lc eq 'Sec-WebSocket-Protocol') {
```

Comment:
shouldn't the right hand 'Sec-WebSocket-Protocol' be written in lowercase if it is compared to the lowercase header name?

Reply:
Ouch...